### ISSUE TITLE
Missing libbpf v0.2 symbols

### ISSUE THREAD

**Issue by danobi** [alexforster/libbpf-sys#15]
I'm getting linker errors when I use the v0.2 libbpf-sys from crates.io:
```
  = note: /bin/ld: /home/daniel/dev/libbpf-rs/target/debug/deps/cargo_libbpf-e1881abc62d2e476.3pzy2gabm1c6gb6f.rcgu.o: in function `cargo_libbpf::btf::btf::Btf::new':
          /home/daniel/dev/libbpf-rs/libbpf-cargo/src/btf/btf.rs:61: undefined reference to `btf__set_endianness'
          collect2: error: ld returned 1 exit status
```

I don't get the error when I use a local checkout of libbpf-sys:
```
$ cat Cargo.toml
[...]

[dependencies]
libbpf-sys = { path = "/home/daniel/dev/libbpf-sys" }

$  nm ./target/debug/build/libbpf-sys-37d4aafae3a38032/out/libbpf.a | rg endian
00000000000016c0 T btf__endianness
00000000000016d0 T btf__set_endianness
```

Since I don't see any special git submodule initialization in the build script, I wonder if the libbpf sources uploaded to crates.io are just whatever you have checked out at time of `cargo publish`.

If that's the case, if it possible you didn't update the libbpf submodule before `cargo publish`ing? Just a guess.

**Comment by alexforster** [comment]
Hmm, that's entirely possible. The ["does it compile and link?"](https://github.com/alexforster/libbpf-sys/blob/6f2974a8d4afc233679367a95fd3f6050d38015f/tests/tests.rs#L18) test [passed](https://travis-ci.org/github/alexforster/libbpf-sys/builds/751488914), but I didn't actually try it in any real codebase. I'll check it out.

**Comment by danobi** [comment]
I think only the symbols introduced in libbpf v0.2 are missing. That test only checks an old symbol so it should still pass.

**Comment by alexforster** [comment]
Published [0.2.0-2](https://crates.io/crates/libbpf-sys/0.2.0-2) and yanked -1. Give it a shot.

**Comment by danobi** [comment]
It works, thanks! And happy holidays :)
